proto/minder/v1/minder.proto

```diff
@@ -2559,70 +2559,73 @@ message ListEvaluationHistoryResponse {
 }
 
 message EvaluationHistory {
-    // entity contains details of the entity which was evaluated.
-    EvaluationHistoryEntity entity = 1;
+    message Entity {
+        // id is the ID of the entity.
+        string id = 1;
 
-    // rule contains details of the rule which the entity was evaluated against.
-    EvaluationHistoryRule rule = 2;
+        // type is the entity type.
+        minder.v1.Entity type = 2;
 
-    // status contains the evaluation status.
-    EvaluationHistoryStatus status = 3;
+        // name is the entity name.
+        string name = 3;
+    }
 
-    // alert contains details of the alerts for this evaluation.
-    EvaluationHistoryAlert alert = 4;
+    message Rule {
+        // name is the name of the rule instance.
+        string name = 1;
 
-    // remediation contains details of the remediation for this evaluation.
-    EvaluationHistoryRemediation remediation = 5;
-}
+        // type is the name of the rule type.
+        string type = 2;
 
-message EvaluationHistoryEntity {
-    // id is the ID of the entity.
-    string id = 1;
+        // profile is the name of the profile which contains the rule.
+        string profile = 3;
+    }
```

Comment:
It feels odd to have `type` fields in both the `Entity` and `Rule` messages that mean different things -- `Entity.type` is an enum (IIRC), while `Rule.type` is effectively the name of a template.  Can we call this `rule_type` to disambiguate?

Reply:
(Also, the lack of `id` here means that `profile` + `name` is the distinguishing set of fields for future follow-up queries.)

Reply:
@evankanderson Are you referring to the rule type ID, or the rule instance ID?

Reply:
I think it would be a rule instance ID if you wanted to be able to do a unique follow-up query.  (i.e. looking for more history for items which don't have any history or have some suspicious changes over a shorter time window)